### PR TITLE
[netcore] Add target for running coreclr tests

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Comment by filipnavara** [comment]
On a related note, do we want to use `corerun` to run the CoreFX tests after this PR lands?

**Comment by marek-safar** [comment]
@filipnavara no, keeping them to run same in the same setup as in corefx repo would be better.

**Comment by filipnavara** [comment]
@marek-safar Both CoreFX and CoreCLR run them using `corerun` AFAIK

**Comment by filipnavara** [comment]
The documentation in CoreFX looks a bit mixed on what is currently used but it looks like it uses `Microsoft.NETCore.TestHost` package which comes from CoreCLR repository and contains pre-build `corerun`.

File: netcore/Makefile [hunk 1]
```diff
@@ -234,6 +239,36 @@ build-test-corefx-%:
 xunit-summary:
 	./xunit-summary.py "corefx/tests"
 
+
+update-tests-coreclr: coreclr/.stamp-tests
+
+coreclr/.stamp-tests:
+	rm -rf coreclr/tests
```
Comment (filipnavara) [inline netcore/Makefile]: The `coreclr` directory is never created on my machine and all the successive step consequently fail.

File: netcore/Makefile [hunk 2]
```diff
@@ -234,6 +239,36 @@ build-test-corefx-%:
...
+	curl -L "$(CORECLR_TESTS)" --output coreclr/tests.zip
+	$(UNZIPCMD) coreclr/tests.zip -C coreclr/
```
Comment (filipnavara) [inline netcore/Makefile]: It downloads a `.zip` file but tries to extract it with `tar` on Linux which fails.

**Pull request merged**